src/language/walker/ruleWalker.ts

```diff
@@ -82,6 +82,14 @@ export class RuleWalker extends SyntaxWalker {
         }
     }
 
+    public createFix(replacements: Replacement[]): Fix {
```

Comment:
Fix is probably redundant. Eventually we should proxy replacements (including createReplacement) to make sure replacements at the same spots are applied in the right order. It turns out sorting is not guaranteed to be stable, but adding an additional ordering counter breaks ties correctly.